src/sass/_header.scss

```diff
@@ -51,4 +60,46 @@ nav {
 		-webkit-text-stroke-width: 0;
 		color: $red-light;
 	}
+
+	span {
+		margin: 0;
+		font-size: 4rem;
+	}
+}
+
+.title-container {
```

Comment:
I've added fill color here so we can also target in dark mode, but I also re-factored just a bit to take advantage of sass nesting:

```
.title-container {
	display: flex;
	.star-right,
	.star-top,
	.star-left {
		position: relative;
		transform: scale(0.6);
		fill: $red-light;
	}
	.star-right {
		top: 23px;
		left: -18px;
	}
	
	.star-top {
		top: -50px;
		left: 68px;
	}
	.star-left {
		top: -2px;
		right: -4px;
	}
	
}
```

```
nav {
	width: 50%;
	max-width: 420px;
	display: flex;
	height: 70px;
	justify-content: space-around;
	align-items: flex-end;
	margin-top: 5px;
	.lines-before,
	.lines-after {
	position: relative;
	transform: scale(0.6);
	stroke: $red-light;
	}
	.lines-before {
		bottom: -8px;
		right: -26px;
	}
	.lines-after {
		bottom: 22px;
		right: 8px;
	}
}

```

Reply:
One thing I love about SASS is how readable it is because of the nesting!

Reply:
I know! It's the 🐝s🦵s!

Reply:
I love this too! I never knew I needed it until I learned it was possible.

Reply:
Woah thank you for refactoring this! The power of sass! 😁